### ISSUE TITLE
Windows fail

### ISSUE THREAD

**Issue by techtonik** [gdamore/proxima5#26]
I am still trying to find a demo that showcases how tcell works. `go get ...` worked to produce the proxima5 binary, but it fails on Windows with `Failed to initialize game: Cannot find data (did you run rebuild.sh?)`. Is it possible to get rid of .sh scripts and make this step a part of get process?

**Comment by techtonik** [comment]
Missing commands:

```
go build mkassets.go spritedata.go leveldata.go properties.go
mkassets -type level l-level1.yml 
mkassets.exe -type sprite s-alien1.yml s-blastwave.yml s-bullet.yml s-cavern1.yml s-complete.yml s-exhaust.yml s-explosion.yml s-gameover.yml s-gantry.yml s-hfence.yml s-pad.yml s-press.yml s-ship.yml s-smexplosion.yml s-thrust.yml s-tinyexplosion.yml
go build .
```

**Comment by gdamore** [comment]
Yes, probably this should be done via go generate.  I used bash on Windows to build this -- from the git installation actually. :-)

**Comment by techtonik** [comment]
I am using ssh on windoze through git installation, but using bash.. that's hackish! =)

**Comment by gdamore** [comment]
I'm not really a Windows developer.  Windows got support for all this stuff mostly as a side effort.  I really don't like the legacy cmd stuff. 

I can do the go generate thing and commit the generated go files if that will help.

**Comment by techtonik** [comment]
If the commands above can be executed during standard go build process with go tools - that will be fine.

**Comment by gdamore** [comment]
You have to do go run...  Because it cannot find things in . on non-Windows platforms.
